front-end/src/App.scss

```diff
@@ -60,10 +60,9 @@
     background-color: lighten($main-bg, 4);
   }
 
-  &__btn-logout:active {
-    border: none;
+  &__btn-logout:active{
```

Comment:
Is it work? Default 'outline' property is installed on 'focus', not 'active'. You can set 'outline: none' to '&__btn-logout' class without using this selectors